service/controller/app/resource/releasemigration/resource.go

```diff
@@ -135,16 +144,17 @@ func (r *Resource) ensureReleasesMigrated(ctx context.Context, k8sClient k8sclie
 		}
 		if len(releases) > 0 {
 			desc := fmt.Sprintf("%d helm v2 releases not migrated", len(releases))
-			r.logger.Log("level", "debug", "message", desc)
+			r.logger.LogCtx(ctx, "level", "debug", "message", desc)
 
 			return microerror.Maskf(executionFailedError, desc)
 		}
+		r.logger.LogCtx(ctx, "level", "debug", "message", "migration completed")
```

Comment:
We do not print any final message after a successful migration, this could be the signal all migration had been finished.